resources/howtos/search_bar_for_docs.md

```diff
@@ -0,0 +1,201 @@
+# Integrate a relevant search bar to your documentation
+
+You might have noticed the search bar in this documentation.
+
+![MeiliSearch docs search bar demo](/tuto-searchbar-for-docs/vuepress-searchbar-demo.gif)
+
+And you might need the same for yours!
+
+This tutorial aims to detail the steps to set up a relevant and powerful search bar in your documentation with MeiliSearch 🚀
+
+1. [Run a MeiliSearch Instance](#run-a-meilisearch-instance)
+2. [Scrap your content](#scrap-your-content)
+3. [Integrate the Search Bar](#integrate-the-search-bar)
+
+## Run a MeiliSearch Instance
+
+First of all, you need your documentation content to be scraped and pushed into a MeiliSearch instance.
+
+This can be done in a few steps. MeiliSearch is open-source and can run on your own server!
+
+Check out this well-explained tutorial about [how to run MeiliSearch in production](/resources/tutorials/running_production.md).
+If you get used to the DigitalOcean droplets, you can easily [create one](https://marketplace.digitalocean.com/apps/meilisearch?action=deploy&refcode=7c67bd97e101).
+
+::: note
+
+The host URL and the API key you will provide in the next steps are the credentials of this MeiliSearch instance.
+
+:::
+
+## Scrap your Content
+
+The Meili team provides and maintains a [scraper tool](https://github.com/meilisearch/docs-scraper) to scrap your website content and automatically index it into MeiliSearch.
```

Comment:
(There are 3 "automatically" in your suggestion @eskombro 🙂)

Will discuss that together or even ask @erlend-sh 😇
Is it better to use technical terms here (`scrap`) or to describe the scraping?

Reply:
I did not see that `scraper tool` was used before the sentence. In which case I think it is a good idea to introduce the concept in this sentence. 

Suggestion on suggestion: I changed indexed by store because it is not the scraper that indexes but MeiliSearch. This keeps the sentence more easy to read.
```suggestion
The Meili team provides and maintains a [scraper tool](https://github.com/meilisearch/docs-scraper) to scrap your website content and store it into MeiliSearch.
```